launchdarkly/metadata.csv

```diff
@@ -0,0 +1,4 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+launchdarkly_relay.connections_env_platformCategory_userAgent,gauge,,connection,,The number of current proxied streaming connections.,0,launchdarkly,LaunchDarkly
+launchdarkly_relay.newconnections_env_platformCategory_relayId_userAgent,gauge,,connection,,The number of streaming connections created.,0,launchdarkly,LaunchDarkly
+launchdarkly_relay.requests_env_method_platformCategory_route_userAgent,gauge,,request,,Number of requests received.,0,launchdarkly,LaunchDarkly

```

Comment:
I think the short names here should better reflect the metric themselves, for ex, the first one could be `relay conn` or something similar.